charts/ceph-csi-rbd/templates/provisioner-deployment.yaml

```diff
@@ -48,7 +48,15 @@ spec:
                     values:
                       - {{ .Values.provisioner.name }}
               topologyKey: "kubernetes.io/hostname"
-{{- end }}
+{{- if .Values.provisioner.affinity }}
+{{ toYaml .Values.provisioner.affinity | indent 8 -}}
+{{- end -}}
+{{- else -}}
+{{- if .Values.provisioner.affinity }}
+      affinity:
```

Comment:
Isn't it the duplicate affinity here? Do you have any test steps and test results for this change?

Reply:
sounds good, i missed `else` statement Thanks for explaning